plugins/nominal-connection-checker/test/index.js

```diff
@@ -64,6 +64,7 @@ function createWorkspace(blocklyDiv, typeHierarchy, blocks) {
 }
 
 document.addEventListener('DOMContentLoaded', function() {
+  window.Blockly = Blockly;
```

Comment:
The playground already does this in https://github.com/google/blockly-samples/blob/master/plugins/dev-tools/src/index.js#L30
Curious why this is necessary here?

Reply:
You're right, I have no clue why I added this haha. Removed!